README.adoc

```diff
@@ -114,7 +114,7 @@ The MicroProfile Metrics API was added as a dependency to your `pom.xml` file. L
 the `org.eclipse.microprofile.metric` group ID. Add this dependency to use MicroProfile Metrics API
 in your code to provide metrics from your microservices.
 
-Navigate to the `start` directory.
+Before you begin, navigate to the `start` directory.
```

Comment:
The location of this sentence doesn't seem to go well with the next sentence that starts with "Next,.." Pls reconsider.